doc/scheduling.md

```diff
@@ -0,0 +1,180 @@
+distributed to all the servers in a group in a fair way.
+
+Scheduler name for use in configuration file: `round-robin`.
+
+
+### Hash Scheduler
+
+Chooses server to schedule request to by hashed key value. Key is built using
+_uri_, _request method_ and _host header_ of the request. In most cases load will
+be distributed among all the servers in group, but situations when single server
+pulls all the load are also possible. Although it is quite improbable, such
+condition is quite stable: it cannot be fixed by adding/removing servers and
+restarting Tempesta FW.
+
+Hash scheduler have an advantages though. It schedules requests of same resources
+to the same servers. That can reduce load of backend servers.
+
+Scheduler name for use in configuration file: `hash`.
+
+
+## Scheduling HTTP sessions to the same server
+
+None of the schedulers is responsible for forwarding requests from the same HTTP
+session to the same server. [Sticky Sessions]() option should be used instead.
+
+
+## Effects caused to schedulers by other configuration options
+
+
+### Limitations
+
+- [HTTP Scheduler](#HTTP Scheduler) must not provide multiple backup groups to
+the same group if [Sticky Sessions]() are enabled.
+
+
+### Effects
+
+- If [Sticky Sessions]() are enabled TempestaFW does its best to reuse last used
+connection for a server. That makes round-robin schedulers to distribute HTTP
+sessions in among servers in server group, but not the requests. Situation when
+one server steals all the heavy sessions is possible. Hash scheduler is not
+expected to work good with sticky sessions, most likely that one server will
+pull most of sessions.
+
+- With [Sticky Sessions]() switching from backup group to main require closing
+all live sessions to servers in backup group.
+
+
+## Troubleshooting
+
+- TempestaFW overloads some of my servers while others in the same group are
+almost unused.
+
+Hash Server scheduler does not guarantee fair load of backend servers and
+can cause situations, when some servers pull most of the load. Round-robin
+scheduler should be used instead. Other configuration options can also affect
+schedulers behaviour, see [Effects caused to schedulers](#Effects caused to schedulers by other configuration options)
+
+- Requests are still forwarded to servers from backup group while main is
+back online.
+
+This happens only if [Sticky Sessions]() are enabled. TempestaFW will stop
+forwarding requests to backup servers as soon as all the HTTP sessions, served
+by backup servers will be closed.
+
+- Request is scheduled to a wrong server group or dropped while all the servers
+are reachable for TempestaFW.
+
+If request is scheduled to a wrong group it is most likely that there is an
+error in HTTP scheduler rules: request suit rule that is closer to the top
+of the list. Put more precise rules before more generic.
+
+Dropping of the message means that dropped request didn't match any rule.
```

Comment:
This seems like a suggestion how to fix some abstract issue due to misconfiguration. I believe documentation like Troubleshooting or FAQ should be written in response to real users experience: when there is some mismatch between the system operation and user expectations.